### PR TITLE
[14.0][MIG] delivery_dropoff_site: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Comment by chafique-delli** [comment]
@sebastienbeau

File: delivery_dropoff_site/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    final_shipping_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Final Recipient",
+        help="It is the partner that will pick up the parcel " "in the dropoff site.",
```
Comment (alexis-via) [inline delivery_dropoff_site/models/stock_move.py:14]: remove " "

File: delivery_dropoff_site/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    final_shipping_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Final Recipient",
+        help="It is the partner that will pick up the parcel " "in the dropoff site.",
```
Comment (alexis-via) [inline delivery_dropoff_site/models/stock_picking.py:14]: remove " "

**Pull request closed without merging**